README.md

```diff
@@ -1,82 +1,28 @@
-
-**SortingHat** is the GrimoireLab component for managing identities and their metadata. It uses a relational database to track all identities found in the repositories, and can use several heuristics and data sources to merge identities and annotate them. For example, it can consume gitdm files with affiliation and merged identities, or its own format to export / import this kind of information. [Learn more about SortingHat](/sortinghat/data.md).
-
-In the usual pipeline, Elk interacts with SortingHat to feed it with all identities found in raw indexes. Later, when producing enriched indexes, it includes the merged identities, affiliation, bot status, etc. produced by SortingHat.
-
-This way, an analysis performed on enriched indexes with the right information coming from SortingHat allows for a correct count of developers and other actors in software development.
-
-### Visualization and analytics
-
-Enriched indexes in GrimoireLab are designed to be visualized and analyzed. They can in principle be fed to any tool that can link to ElasticSearch indexes, and of course custom code can also be produced for the analysis. But GrimoireLab already provides some components to simplify this task:
-
-* **Kibiter**, a soft fork of [Kibana](https://www.elastic.co/products/kibana), or Kibana itself, can be used to visualize data in actionable dashboards. Kibana is the standard dashboarding system for ElasticSearch, and Kibiter offers some small, but useful, added functionality for it. Both can be used directly with enriched indexes. Building visualizations and dashboards with them is easy. [Learn more about Kibiter](/basics/scenarios.md)
-
-* **Sigils** is a set of visualizations and dashboards that can be used to produce default dashboards for the enriched indexes. They are loaded in Kibiter or Kibana with Kidash, also a GrimoireLab tool, and provide the means for actionable inspection, drill down, and filtering of many facets of the data in the analyzed repositories. [Learn more about Sigils](https://github.com/chaoss/grimoirelab-sigils).
-
-* **Manuscripts** is a tool to produce summary tables and documents with analytics about enriched indexes. The tables are available in CSV format so they can be imported into spreadsheets or other programs. Documents can be produced as PDF files, ready to be delivered to decision makers, showing relevant aspects of the analyzed project. [Learn more about Manuscripts](/manuscripts/first.md).
-
-----
+```bash
+$ cd grimoirelab-tutorial
+$ bundle
+```
 
+Then [start serving the site locally](https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll):
 
-![](assets/Chaoss_Logo_Pantone1.png)
+```bash
+$ bundle exec jekyll serve
+```
 
-GrimoireLab is a [CHAOSS](http://chaoss.community) project.
+Open the site in the browser of your choice: http://127.0.0.1:4000
```

Comment:
These installation steps should better go to CONTRIBUTING.md file. WDYT?

Reply:
Yes. We can add them. We already have a section that redirects to CONTRIBUTING.md so we can update the latter accordingly

Reply:
I have a small doubt. 
The installation steps should be added [here](https://github.com/chaoss/grimoirelab-tutorial/blob/master/contributing.md) right or did you mean [here](https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md) since you mentioned contributing.md as CONTRIBUTING.md ?

Reply:
I was referring to this https://github.com/chaoss/grimoirelab-tutorial/blob/master/contributing.md#how-to-contribute because it has steps for the old tutorial.

Reply:
You can include this in the next PRs.